docs/PRIVACY.md

```diff
@@ -0,0 +1,127 @@
+
+### Account Information
+
+You may update, correct or delete information about you at any time by logging into your online account. If you wish to delete your account, please email us at admin@compmodels.com, but note that we may retain certain information as required by law or for legitimate business purposes. We may also retain cached or archived copies of information about you for a certain period of time.
+
+### Cookies
+
+Most web browsers are set to accept cookies by default. If you prefer, you can usually choose to set your browser to remove or reject browser cookies. Please note that if you choose to remove or reject cookies, this could affect the availability and functionality of our Services.
+
+### Promotional Communications
+
+You may opt out of receiving promotional communications from COMP by following the instructions in those communications. If you opt out, we may still send you non-promotional communications, such as those about your account or our ongoing business relations.
+
+### Access, Erasure and Correction
+
+The laws in the European Economic Area and certain jurisdictions outside the United States, require us to provide users (to the extent those laws apply to such users) with information related to access, erasure and correction. Upon request COMP will provide you with information about whether we hold any of your personal information. If you would like to review, delete or update your information, you may contact us using the contact information below. We will permit you to correct, amend, or delete information that is demonstrated to be inaccurate. We will respond to your request within a reasonable timeframe. Please note, because of the way we maintain certain services, after you delete or amend your information, residual copies may take a period of time before they are deleted from our active servers and may remain in our backup systems. You will need to provide sufficient identifying information, such as your name and email address and possible additional identifying information as a security precaution.
+
+### Changes to this Privacy Policy
+
+We may change this Privacy Policy from time to time. If we make changes, we will notify you by revising the date at the bottom of the policy and, in some cases, we may provide you with additional notice (such as adding a statement to our homepage or sending you a notification). We encourage you to review the Privacy Policy whenever you access our Services to stay informed about our information practices and the ways you can help protect your privacy.
+
+Contact Information
+To ask questions or comments about this Privacy Policy or our privacy practices please email us at admin@compmodels.com or write us at Compute Tooling, Inc. 213 E Highland Ave., Atlantic Highlands, NJ 07716. 
+
+This version of the Privacy Policy was last updated on December 22, 2018 and historic versions are archived here: https://github.com/comp-org/comp/commits/master/docs/PRIVACY.md.
```

Comment:
I think the link should be: https://github.com/comp-org/comp/blob/master/docs/PRIVACY.md

Reply:
Your link takes me to the latest, but I want to link to the version archives, which is the link I have. See, https://github.com/comp-org/comp/commits/master/docs/TERMS.md and https://github.com/comp-org/comp/blob/master/docs/TERMS.md

Reply:
Ah, I see. That makes sense.